lib/buffer.js

```diff
@@ -54,6 +55,11 @@ function alignPool() {
   }
 }
 
+const bufferDeprecationWarning =
+    deprecate(() => {}, 'Using Buffer without `new` will soon stop working. ' +
+      'Use `new Buffer`, or preferably ' +
+      'Buffer.from, Buffer.allocUnsafe or Buffer.alloc instead.');
```

Comment:
probably do not mention `allocUnsafe`?

if people need that they will find it I think :)